src/components/Task/Drawer/CommentList/CommentList.tsx

```diff
@@ -85,7 +93,7 @@ const TaskDrawerCommentList = ({
                     <TaskDrawerCommentListItem
                       comment={comment}
                       reverse={comment.me}
-                      nextComment={data.task.comments.nodes[result.length + 1]}
+                      nextComment={data.task.comments?.nodes[result.length + 1]}
```

Comment:
You didn't write this but it seems a little odd. I guess it's fine, I was just trying to think if there was a better way to do it. You could maybe make two arrays, shift the second array left one (by removing the first element), zip them together so you've got pairs of the comment and the next comment and then map but I think that's worse 😄 

Does this help make it slightly simpler? 🤷‍♂️ 
```suggestion
                      nextComment={nodes[index + 1]}
```